x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/use_supported_response_action_types.tsx

```diff
@@ -28,9 +28,9 @@ export const useSupportedResponseActionTypes = () => {
 
   const userHasPermissionsToExecute = useMemo(
     () => ({
-      endpoint: canIsolateHost,
+      endpoint: canIsolateHost || canKillProcess || canSuspendProcess,
```

Comment:
This should be also checking the feature flag, no?

Reply:
Hmm, I think it might, but doesn't really have to. Currently the `automatedProcessActionsEnabled` is being checked in the actions dropdown, and while scheduling action call. 
Do you think we could make use of more checks?